examples/rapids-image/.ipynb_checkpoints/Dockerfile-checkpoint

```diff
@@ -0,0 +1 @@
+FROM rapidsai/rapidsai
```

Comment:
Could we remove the `ipynb_checkpoints` directory from the PR? Bonus points if you can update the `.gitignore` to omit `.ipynb_checkpoints` all together.